flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-icons-check": true,
+    "only-arches": ["x86_64", "aarch64"]
```

Comment:
Just remove the whole file. There is no other arch supported at the time.

Reply:
fixed